bitnami/cassandra/values.yaml

```diff
@@ -770,6 +783,15 @@ metrics:
     http: 8080
     jmx: 5555
 
+  ## Cassandra ports to be exposed as hostPort
```

Comment:
```suggestion
  ## Metrics ports to be exposed as hostPort
```

Reply:
Just a minor comment, everything else LGTM

Reply:
Fixed in fac1935